tests/gpflow/conditionals/test_multioutput.py

```diff
@@ -317,29 +317,104 @@ def test_sample_conditional_mixedkernel():
     )
 
 
-@pytest.mark.parametrize(
-    "func, R",
-    [
-        (fully_correlated_conditional_repeat, 5),
-        (fully_correlated_conditional_repeat, 1),
-        (fully_correlated_conditional, 1),
-    ],
+@pytest.fixture(
+    name="q_sqrt_factory", params=[lambda _, __: None, lambda LM, R: tf.eye(LM, batch_shape=(R,))]
```

Comment:
this seems specific to the fully-correlated case,
```suggestion
    name="fully_correlated_q_sqrt_factory", params=[lambda _, __: None, lambda LM, R: tf.eye(LM, batch_shape=(R,))]
```
to make this clear (and avoid accidental misuse in the future)?